tests/specs/lemmas.k

```diff
@@ -209,6 +211,11 @@ module LEMMAS
     syntax Bool ::= #notKLabel ( K , String ) [function]
     rule #notKLabel(T, L) => #getKLabelString(T) =/=String L orBool #isVariable(T)
 
+    rule N <=Int maxInt(P, Q) => true requires N <=Int P orBool N <=Int Q  [simplification]
+    rule minInt(P, Q)         => P    requires P <=Int Q                   [simplification]
+
     rule 0 <=Int #sizeWordStack ( _ , _ ) => true [smt-lemma]
+    rule 0 <=Int #sizeByteArray ( _ )     => true [smt-lemma]
+    rule 0 <=Int lengthBytes ( _ )        => true [smt-lemma]
```

Comment:
i think `rule 0 <=Int lengthBytes ( _ )        => true [smt-lemma]` is not needed because:
1. if the argument is concrete, we will have a concrete value.
2. if is symbolic, we will have `#sizeByteArray` instead o f`lengthBytes`.